test/jit/test_python_bindings.py

```diff
@@ -0,0 +1,38 @@
+import torch
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == "__main__":
+    raise RuntimeError(
+        "This test file is not meant to be run directly, use:\n\n"
+        "\tpython test/test_jit.py TestPythonBindings\n\n"
+        "instead."
+    )
+
+
+class TestPythonBindings(JitTestCase):
+    def test_get_python_cu(self):
+        # tests get_python_cu
+        @torch.jit.script
+        def test_get_python_cu_fn(x: torch.Tensor):
+            return 2 * x
+
+        cu = torch._C._jit_get_python_cu()
+        self.assertTrue(
+            "test_get_python_cu_fn" in (str(fn.name) for fn in cu.get_functions())
```

Comment:
Consider using `find_function` here if you haven't already.

Reply:
I wanted to see that `get_functions` works, too.